helix-term/src/commands.rs

```diff
@@ -1836,6 +1839,11 @@ mod cmd {
         });
         let future = doc.format_and_save(fmt);
         cx.jobs.add(Job::new(future).wait_before_exiting());
+
+        if refresh_ls {
```

Comment:
This var isn't needed, just use `doc.path().is_some()`

Reply:
Actually, we already bail on L1826 if there's no path, so this call should just always happen

Reply:
Can't you just use `path.is_some()` here then?